Shared/Utils.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  Utils.swift
+//  Client
+//
+//  Created by Atul Aggarwal on 8/8/15.
+//  Copyright (c) 2015 Mozilla. All rights reserved.
+//
+
+import AVFoundation
+import UIKit
+
+public func getUserAgent() -> String {
```

Comment:
Eventually that'll let us split out individual pieces of functionality here for testing, amongst other things.